### PR TITLE
Add lightshot info

### PR DESCRIPTION
**Pull request by NyCodeGHG**

### CODE REVIEW

File: src/main/kotlin/de/nycode/bankobot/listeners/LightshotListener.kt [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+package de.nycode.bankobot.listeners
+
+import de.nycode.bankobot.utils.Embeds
+import dev.kord.core.Kord
+import dev.kord.core.behavior.reply
+import dev.kord.core.event.message.MessageCreateEvent
+import dev.kord.core.on
+
+private val lightshotRegex by lazy { "(?:http[s]?://)?(?:prnt\\.sc|prntscr.com)".toRegex() }
```
Comment (DRSchlaubi) [inline src/main/kotlin/de/nycode/bankobot/listeners/LightshotListener.kt:34]: no regex101.com link?

**Review by DRSchlaubi — CHANGES_REQUESTED** [review]

File: src/main/kotlin/de/nycode/bankobot/listeners/LightshotListener.kt (hunk 1 above)
Comment (NyCodeGHG) [reply src/main/kotlin/de/nycode/bankobot/listeners/LightshotListener.kt:34]: i created this using regex101, but i forgot to link it

File: src/main/kotlin/de/nycode/bankobot/listeners/LightshotListener.kt [hunk 2]
```diff
@@ -31,6 +31,7 @@ import dev.kord.core.behavior.reply
 import dev.kord.core.event.message.MessageCreateEvent
 import dev.kord.core.on
 
+// https://regex101.com/r/JEHxwG/1
 private val lightshotRegex by lazy { "(?:http[s]?://)?(?:prnt\\.sc|prntscr.com)".toRegex() }
```
Comment (DRSchlaubi) [inline src/main/kotlin/de/nycode/bankobot/listeners/LightshotListener.kt:35]: Escapte the .com `.` because right now it matches anything

**Review by DRSchlaubi — CHANGES_REQUESTED** [review]

File: src/main/kotlin/de/nycode/bankobot/listeners/LightshotListener.kt (hunk 2 above)
Comment (NyCodeGHG) [reply src/main/kotlin/de/nycode/bankobot/listeners/LightshotListener.kt:35]: oh nvm i thought i fixed that

**Review by DRSchlaubi — APPROVED** [review]

**Pull request merged**